torch/csrc/distributed/autograd/autograd.cpp

```diff
@@ -0,0 +1,24 @@
+#include <torch/csrc/distributed/autograd/autograd.h>
+#include <ATen/record_function.h>
+
+namespace torch {
+namespace distributed {
+namespace autograd {
+
+void backward(
+    int64_t context_id,
+    const variable_list& roots,
```

Comment:
This used to be a vector of Tensors instead of Variables, and then explicitly creating a vector of Variables. Will this cause any problem? Or are all call sites supposed to pass in Variables?

Reply:
I think all the callsites should pass in variables to begin with, although right now tensor/variable are the same thing.